avaframe/out3Plot/outQuickPlot.py

```diff
@@ -580,3 +580,64 @@ def plotContours(contourDict, resType, thresholdValue, pathDict, addLegend=True)
     # save and or plot
     outFileName = pathDict['plotScenario'] + '_ContourLines'
     pU.saveAndOrPlot(pathDict, outFileName, fig)
+
+
+def plotAllContours(avaDir, modName, resType, level, specDir=''):
+    """ Create a plot of resType level contour lines for all resType files found in
+        avaDir/Outputs/modName/peakFiles or specDir optional
+        file format needs to be of type _resType.asc
+
+        Parameters
+        ------------
+        avaDir: str or pathlib path
+            path to avalanche directory
+        modName: str
+            name of computational module
+        resType: str
+            result variable type
+        level: float
+            contour line level
+        specDir: str or pathlib path
```

Comment:
minor: what does specDir mean? Why not peakFileDir or something more descriptive?